pages/data.py

```diff
@@ -58,6 +59,13 @@ def render_content(tab, store_uuids, store_trips):
             col['label'] for col in perm_utils.get_allowed_named_trip_columns()
         )
         has_perm = perm_utils.has_permission('data_trips')
+    elif tab == 'tab-demographics-datatable':
+        data = store_demographics["data"]
+        columns = list(data[0].keys())
+        for column in perm_utils.permissions.get("data_demographics_columns_exclude", []):
+            columns.discard(column)
```

Comment:
**(future)** Is there a reason that you discard here (in `data.py`) while the other calls to `discard` are in `permissions.py`?
https://github.com/search?q=repo%3Ae-mission%2Fop-admin-dashboard%20discard&type=code